app/models/game.rb

```diff
@@ -9,3 +9,57 @@ class Game < ApplicationRecord
   validates :white_player, presence: true
   validates :black_player, presence: true
 end
+
+class Game < ApplicationRecord
+  belongs_to :white_player, class_name: 'User'
+  belongs_to :black_player, class_name: 'User'
+  validates :name, presence: true
+  validates :result, inclusion: {
+    in: %w(in_progress white_win black_win draw),
+    message: '%{value} is not a valid result'
+  }
+  validates :white_player, presence: true
+  validates :black_player, presence: true
+end
+#game rules of chess
+def initialize_board
+  #white
+  (1..8).each do |index|
+    Pawn.create(
+      x_position: 1,
+      y_position: index,
```

Comment:
`x_position` and `y_position` are reversed here; the pawns should be arrayed across `y_position: 2`.